src/OpenTK/Platform/MacOS/Cocoa/NSDragOperation.cs

```diff
@@ -2,6 +2,10 @@
 
 namespace OpenTK.Platform.MacOS
 {
+    // Values for enum : https://developer.apple.com/documentation/appkit/nsdragoperation?language=objc
```

Comment:
Please use XML comments instead. They can be parsed by the doc generator and will be included in documentation files - normal comments will not.